package.json

```diff
@@ -8,17 +8,48 @@
     "supabase:stop": "supabase stop",
     "db:migrate": "supabase db reset",
     "db:remote:migrate": "supabase db push",
-    "db:remote:seed": "ts-node supabase/seed-remote.ts"
+    "db:remote:seed": "ts-node supabase/seed-remote.ts",
+    "build": "nest build",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "prepare": "npm run build",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest"
   },
   "keywords": [],
   "author": "Black Flower (https://github.com/blkflower)",
   "license": "ISC",
   "devDependencies": {
+    "@supabase/supabase-js": "^1.35.7",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^14.14.20",
+    "@types/passport-jwt": "^3.0.10",
+    "@types/passport-strategy": "^0.2.35",
     "@types/pg": "^8.10.3",
+    "@typescript-eslint/eslint-plugin": "^4.12.0",
+    "@typescript-eslint/parser": "^4.12.0",
+    "eslint": "^7.17.0",
+    "eslint-config-prettier": "^7.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.1.3",
+    "jest": "^26.6.3",
+    "passport-jwt": "^4.0.1",
+    "passport-strategy": "^1.0.0",
+    "prettier": "^2.2.1",
     "supabase": "^1.100.1",
-    "ts-node": "^10.9.1"
+    "ts-jest": "^26.4.4",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.1.3"
   },
   "dependencies": {
+    "@nestjs/common": "^10.0.0",
+    "@nestjs/config": "^3.1.1",
+    "@nestjs/core": "^10.0.0",
+    "@nestjs/passport": "^10.0.2",
```

Comment:
@philip-calape Do we still need passport if we're using Supabase for authentication?

Reply:
Hello @arnellebalane, yes we still need passport for our AuthGuard and strategy and for connecting in nestjs code. Supabase is used for our underlying authentication